core/startos/src/service/service_effect_handler.rs

```diff
@@ -1037,14 +1057,15 @@ impl ValueParserFactory for Status {
 #[command(rename_all = "camelCase")]
 #[ts(export)]
 struct SetMainStatus {
-    status: Status,
+    status: SetMainStatusStatus,
 }
 async fn set_main_status(context: EffectContext, params: SetMainStatus) -> Result<Value, Error> {
     dbg!(format!("Status for main will be is {params:?}"));
     let context = context.deref()?;
     match params.status {
-        Status::Running => context.started(),
-        Status::Stopped => context.stopped(),
+        SetMainStatusStatus::Running => context.started(),
+        SetMainStatusStatus::Stopped => context.stopped(),
+        SetMainStatusStatus::Starting => context.stopped(),
```

Comment:
Why add a varient if just treating as stopped?

Reply:
To indicate a state, that it is trying to getting things started. Could be more usefull or not, but for now just not doing anything with this information until I talk with you.